src/SampleWebApp/Controllers/BooksMvcController.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    public class BooksMvcController : Controller
+    {
+        private BookDataContext _context;
+
+        public BooksMvcController(BookDataContext context)
+        {
+            _context = context;    
+        }
+
+        // GET: BooksMvc
+        public IActionResult Index()
+        {
+            var bookDataContext = _context.Book.Include(b => b.Author);
```

Comment:
@prafullbhosale - does the dialog let you customize the DbSet property name when generating the code? This might be something we want to consider. IIRC we didn't support this in the past, but it might make sense to have for EF Core